### ISSUE TITLE
1.5.2 does not support cross references within pdfs

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#321]
```
What steps will reproduce the problem?
1.Take a pdf containing cross-refs, like the one attached.
2.Open it with EBookDroid.
3.Tap one of the links (there's one on page 3 for example).

What is the expected output?
The app goes where the link points (a page of this very pdf in our case).

What do you see instead?
Nothing happens. Tapping has no effect.

What version of the product are you using?
1.5.2

On what operating system?
Android 2.3.4

What device are you using?
Galaxy Ace (GT-S5830)

Please provide any additional information below.
The feature works all right on the 1.4.2-version of EBookDroid.
The problem has nothing to do with a specific file - it remains the same on all 
of them.

```

Original issue reported on code.google.com by `an7onloo...@gmail.com` on 22 Sep 2012 at 9:22
- Merged into: #307

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
I haven't mentioned that it had already been fixed in 1.5.3-dev-r1640.apk 
Mea culpa.
```

Original comment by `an7onloo...@gmail.com` on 22 Sep 2012 at 9:32

**Comment by GoogleCodeExporter** [comment]
Original comment by `Alexander.V.Kasatkin@gmail.com` on 23 Sep 2012 at 10:46
- Changed state: **Duplicate**
- Added labels: **pdflinks**
